test/CMakeLists.txt

```
@@ -5,6 +5,8 @@ include(ExternalProject)
 find_package(SystemBLAS REQUIRED)
 
 set(SYCLBLAS_TEST ${CMAKE_CURRENT_SOURCE_DIR})
+set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations -Wno-deprecated-declarations -Wno-shorten-64-to-32 -Wno-cast-align") 
```

```suggestion
set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations -Wno-shorten-64-to-32 -Wno-cast-align") 
```
A flag is repeated twice.